detect_meteors.py

```diff
@@ -7,10 +7,16 @@
 import scipy as sp
 import scipy.stats
 import os
+from collections import namedtuple
+import math
+from time_utils import datetime_to_float
 
 import digital_rf_hdf5 as drf
 import digital_metadata as dmd
 import TimingModeManager
+from module_skeleton import *
```

Comment:
Better to explicitly import the functions needed from module_skeleton, or import the whole module and access the functions with the dot operator. With * imports, you don't know what you're importing or if it will clash with names already in the namespace.

We also need a better name than module_skeleton that actually describes the functions it contains, like meteor_processing.